React/CoreModules/RCTActionSheetManager.mm

```diff
@@ -37,22 +37,30 @@ @interface RCTActionSheetManager () <
 @implementation RCTActionSheetManager
 #if TARGET_OS_OSX // [macOS
 {
-  // Use NSMapTable, as UIAlertViews do not implement <NSCopying>
-  // which is required for NSDictionary keys
+  /* Unlike UIAlertAction (which takes a block for it's action), NSMenuItem takes a selector.
+   * That selector no longer has has access to the method argument `callback`, so we must save it
+   * as an instance variable, that we can access in `menuItemDidTap`. We must do this as well for 
+   * `failureCallback` and `successCallback`.
+   */
   NSMapTable *_callbacks;
-  NSArray<NSSharingService*> *_excludedActivities;
-  NSString *_sharingSubject;
   RCTResponseSenderBlock _failureCallback;
   RCTResponseSenderBlock _successCallback;
+  NSArray<NSSharingService*> *_excludedActivities;
```

Comment:
nit: why we moving lcoation?

Reply:
The updated comment on lines 40-44 refer to all the callback ivars. So I wanted to group them together.